### PR TITLE
feat: Navigation Done

### PR DESCRIPTION
**Pull request by maitrakhatri**
Please review the navigation component

Screenshot 2022-02-06 at 00-11-06 navigation

Here's the documentation for it

Screenshot 2022-02-06 at 00-12-06 gozo UI Components

### CODE REVIEW

File: CSS/Components/navigation/navigation.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <div class="navbar-links">
+
+            <a href="#"><img src="../../../assets/download_white_36dp.svg" alt="download"></a>
+            <a href="#"><img src="../../../assets/print_white_36dp.svg" alt="print"></a>
+            <a href="#"><img src="../../../assets/bookmark_white_36dp.svg" alt="bookmark"></a>
+
+        </div>
+        
+    </nav>
+
```
Comment (Naveenchandar) [inline CSS/Components/navigation/navigation.html]: You can remove the extra spaces and format the file

File: docs/components.html [hunk 2]
```diff
@@ -762,6 +762,41 @@ <h3 class="sub-section-title">Shadow</h3>
...
+
+                </div>
+
+        </div>
```
Comment (Naveenchandar) [inline docs/components.html]: Remove the extra space and format the file

File: CSS/Components/navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+.navbar-header {
+    display: flex;
+    gap: 1rem;
+    padding: 1rem;
+    align-items: center;
+    margin-right: auto;
+}
+
+.navbar-links {
+    display: flex;
```
Comment (Naveenchandar) [inline CSS/Components/navigation/navigation.css:14]: If flex is used often, you can move the flex properties to utility file.

**Review by ianuragsharma — APPROVED** [review]
Add those changes suggested by Naveen chandar,rest looks good to me.

**Comment by maitrakhatri** [comment]
> Add those changes suggested by Naveen chandar,rest looks good to me.

done

File: docs/components.html (hunk 2 above)
Comment (maitrakhatri) [reply docs/components.html]: done

File: CSS/Components/navigation/navigation.html (hunk 1 above)
Comment (maitrakhatri) [reply CSS/Components/navigation/navigation.html]: done

**Pull request merged**